desmume/src/filter/xbrz.cpp

```diff
@@ -478,7 +478,7 @@ const bool sPixDoLineBlend(const Kernel_3x3& ker, const char blend, const xbrz::
 		return true;
 	
 	//make sure there is no second blending in an adjacent rotation for this pixel: handles insular pixels, mario eyes
-	if (getTopR(blend) != BLEND_NONE && !sPixEQ<ColorDistance>(e, g, cfg)) //but support double-blending for 90� corners
+	if (getTopR(blend) != BLEND_NONE && !sPixEQ<ColorDistance>(e, g, cfg)) //but support double-blending for 90� corners

```

Comment:
it seems your editor automatically replaces characters with your local windows charset resulting in bogus output for regular utf8 users. configure it so it doesn't touch stuff *you* didn't touch. but if you did it yourself: we don't want gratuitous whitespace or other changes in stuff unrelated to the PR's core functionality. that just produces noise for reviewers and later people reading the commit log.